cli/internal/globby/globby.go

```diff
@@ -1,15 +1,21 @@
 package globby
 
 import (
-	"github.com/vercel/turborepo/cli/internal/fs"
-
+	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/bmatcuk/doublestar/v4"
+	"github.com/spf13/afero"
 )
 
+var _osFS = afero.NewOsFs()
```

Comment:
Not going to stress about this now, but in general `_` is reserved for unused arguments.

Reply:
Just following the Uber style guide! That's why I made this change. 🤷‍♂️

https://github.com/uber-go/guide/blob/master/style.md#prefix-unexported-globals-with-_